cmake/modules/PrecompiledHeader.cmake

```diff
@@ -171,19 +171,24 @@ function(add_precompiled_header _target _input)
     set(_pch_flags_file "${_pch_binary_dir}/compile_flags.rsp")
     export_all_flags("${_pch_flags_file}")
     set(_compiler_FLAGS "@${_pch_flags_file}")
+    if(CMAKE_CXX_STANDARD EQUAL 11)
+        set(_cpp_standard_FLAG "-std=c++11")
```

Comment:
without above change you'll get -std=c++11 for pch and -std=gnu++11 for everything else